migrations/20170816090954-add_bots_column.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+'use strict';
+const TBL = 'RoomTypes';
+
+module.exports = {
+  up: function (qi, Sequelize) {
+    let attr;
+    return qi.describeTable(TBL)
+    .then((attributes) => {
+      attr = attributes;
+      if (!attr.hasOwnProperty('bots')) {
+        return qi.addColumn(TBL, 'bots', {
+          type: Sequelize.ARRAY(Sequelize.STRING),
+          allowNull: true,
+        });
+      } else {
```

Comment:
since you're doing a return on line 19, you do not need to wrap the return on line 24 inside an "else" block